src/Map/composer.json

```
@@ -0,0 +1,48 @@
+{
+    "name": "symfony/ux-map",
+    "type": "symfony-bundle",
+    "description": "Easily embed interactive maps in your Symfony application",
+    "keywords": [
+        "symfony-ux"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Hugo Alliaume",
+            "email": "hugo@alliau.me"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Symfony\\UX\\Map\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Symfony\\UX\\Map\\Tests\\": "tests/"
+        }
+    },
+    "require": {
+        "php": ">=8.1",
```

```suggestion
        "php": ">=8.2",
```

To be honest i'd set it to PHP 8.3 😅 

https://symfony.com/releases/7.2

---

Yep, I went for it for the 2nd iteration :)